src/reanimated2/PropAdapters.ts

```diff
@@ -1,5 +1,5 @@
 import { addWhitelistedNativeProps } from '../ConfigHelper';
-import type { AdapterWorkletFunction } from './commonTypes';
+import type { __AdapterWorkletFunction } from './commonTypes';
```

Comment:
What's the idea behind prefixing these types with double underscore? Internal-only type? Temporary type?

Reply:
I renamed old, inaccurate types so now they have a double underscore. Since their names were good (most of the time) I wanted a clear distinction to make the transition easier for me.